src/metrics-logger.ts

```diff
@@ -27,30 +27,20 @@ export class MetricsLogFactory {
 
     createLogger() {
         return (request: Request, response: Response, route: Route) => {
-            // Ignore requests without timers, e.g. OPTIONS
-            if (!request.timers) {
-                return;
+            let latency = response.header('Response-Time');
+
+            if (typeof (latency) !== 'number') {
+                latency = Date.now() - request.time();
             }
 
-            let timer = request.timers.find((item) => {
-                return item.name === route.name || item.name === DEFAULT_KEY_NAME;
```

Comment:
Note: alternatively we could change this to
```js
return item.name === DEFAULT_KEY_NAME || (route && item.name === route.name);
```

But i think this metric should be capturing the entire request time, not just the time spent in a specific handler.